lib/core/modules/coderunner/codeRunner.js

```diff
@@ -41,6 +41,19 @@ class CodeRunner {
       if (!cb) {
         cb = function() {};
       }
+      const awaitIdx = code.indexOf('await');
+      if (awaitIdx > -1) {
+        if (awaitIdx < 2) {
+          let end = code.length;
+          if (code[end - 1] === ';') {
+            end--; // Remove the `;` because we add function calls
+          }
+          code = code.substring(5, end); // remove await keyword
+          code += '.then(console.log).catch(console.error);'; // Add promise catch
+        } else {
+          code = `(async function() {${code}})();`;
+        }
+      }
       let result = RunCode.doEval(code);
```

Comment:
To not print the result of the promise and to avoid every `console.log`
Did you consider doing (Without `else`  :bomb: ):
```
if (result instanceof Promise) {
  return result.then((value) => cb(null, value)).catch((error) => cb(error));
}
cb(null, result);
}
```
This is untested

Reply:
It worked. You're a genius